mappings/net/minecraft/client/render/model/BakedQuadFactory.mapping

```diff
@@ -1,5 +1,79 @@
 CLASS net/minecraft/class_796 net/minecraft/client/render/model/BakedQuadFactory
+	CLASS class_797 UVMapper
+		METHOD method_3469 remap (Lnet/minecraft/class_787;)Lnet/minecraft/class_787;
+			ARG 1 texture
+		METHOD method_3470 remap (FFFF)Lnet/minecraft/class_787;
+			ARG 1 startU
+			ARG 2 startV
+			ARG 3 endU
+			ARG 4 endV
+	FIELD field_4258 ORIGIN Lnet/minecraft/class_796$class_797;
+	FIELD field_4259 MAX_SCALE F
+	FIELD field_4260 MIN_SCALE F
+	FIELD field_4261 ROTATE_270 Lnet/minecraft/class_796$class_797;
+	FIELD field_4262 ROTATE_180 Lnet/minecraft/class_796$class_797;
+	FIELD field_4263 ROTATE_90 Lnet/minecraft/class_796$class_797;
+	FIELD field_4264 TEXTURE_ROTATIONS [Lnet/minecraft/class_796$class_797;
 	METHOD method_3454 uvLock (Lnet/minecraft/class_787;Lnet/minecraft/class_2350;Lnet/minecraft/class_1086;)Lnet/minecraft/class_787;
+		ARG 1 texture
+		ARG 2 direction
+		ARG 3 rotation
+	METHOD method_3455 transformCorner (Lnet/minecraft/class_1160;Lnet/minecraft/class_2350;ILnet/minecraft/class_1086;)I
+		ARG 1 rotationVector
+		ARG 2 direction
+		ARG 3 corner
+		ARG 4 rotation
+	METHOD method_3456 getLightBrightness (Lnet/minecraft/class_2350;)F
```

Comment:
Is `Light` necessary? What other kinds of brightnesses are there?

Reply:
There is the base texture brightness and the directional ambient brightness that changes between different sides. This was specifically dealing with the latter.